tools/bin/pr_body.sh

```diff
@@ -9,7 +9,10 @@ GIT_REVISION=$(git rev-parse HEAD)
 
 echo "Changelog:"
 echo
-PAGER=cat git log v${PREV_VERSION}..${GIT_REVISION} --oneline --decorate=no
+# TODO: I'm not sure what the PAGER variable here is for
+PAGER=$(git log "v${PREV_VERSION}..${GIT_REVISION}" --oneline --decorate=no)
+echo "${PAGER}"
+export PAGER
```

Comment:
The PAGER consists of commits that are included in the new version.

It is put into the PR description so people can easily copy them into a new release. An example here: https://github.com/airbytehq/airbyte/pull/9119

I think we only need to `echo` it, not `export it.

Reply:
Yeah I got the changelog part, I am just confused on why the `PAGER` variable even exists. Doesnt this script just need to STDOUT the changelog? I didnt see the need for a variable anywhere. This is how its called:

```
      - name: Get PR Body
        id: pr_body
        env:
          PREV_VERSION: ${{ steps.old_version.outputs.OLD_VERSION }}
        run: |
          chmod +x tools/bin/pr_body.sh
          body=$(./tools/bin/pr_body.sh)
```

its just gradding STDOUT from running the script in a subshell

Reply:
However I dont want to break something Im not aware of so maybe ill just disable the shellcheck thing that was being thrown

Reply:
Right, `echo` it into stdout should be sufficient.

Reply:
Cool, Ill just remove the variable